.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -63,9 +63,11 @@ body:
     attributes:
       label: Installation method
       options:
-        - Docker
+        - Docker - official image (ghcr.io/paperless-ngx/paperless-ngx)
```

Comment:
Maybe better to not mention one particular source, because there is also the official https://hub.docker.com/r/paperlessngx/paperless-ngx?

The long text makes the dropdown menu look a bit weird, too.
![Untitled](https://user-images.githubusercontent.com/9874850/173616603-2a06c50d-aff2-466b-b7df-7dc0f6d27464.png)

Reply:
Sure!